### PR TITLE
LPS-139409

### PR DESCRIPTION
**Pull request by alinedoleron**
Ticket: https://issues.liferay.com/browse/LPS-139409

https://user-images.githubusercontent.com/26491796/177884568-b25ab056-f7b8-4115-99f7-cc3d68876630.mp4

### CODE REVIEW

**Comment by alinedoleron** [comment]
ci:test:sf

**Comment by alinedoleron** [comment]
ci:test:relevant

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/list_type_definitions/edit_list_type_definition.jsp [hunk 1]
```diff
@@ -143,11 +143,15 @@ ViewListTypeEntriesDisplayContext viewListTypeEntriesDisplayContext = (ViewListT
...
+					const parentWindow = Liferay.Util.getOpener();
 
 					setTimeout(() => {
-						const parentWindow = Liferay.Util.getOpener();
 						parentWindow.Liferay.fire('close-side-panel');
 					}, 1500);
+
+					setTimeout(() => {
+						parentWindow.location.reload();
+					}, 2000);
```
Comment (miltonmc) [inline modules/apps/object/object-web/src/main/resources/META-INF/resources/list_type_definitions/edit_list_type_definition.jsp:154]: Use a single `setTimeout()`:
```tsx
					setTimeout(() => {
						const parentWindow = Liferay.Util.getOpener();
						parentWindow.Liferay.fire('close-side-panel');
						parentWindow.location.reload()
					}, 1500);
```

**Review by miltonmc — CHANGES_REQUESTED** [review]
Just a small tweak:

File: modules/apps/object/object-web/src/main/resources/META-INF/resources/list_type_definitions/edit_list_type_definition.jsp (hunk 1 above)
Comment (alinedoleron) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/list_type_definitions/edit_list_type_definition.jsp:154]: Hi @miltonmc! Unfortunately it doesn't work. :confused:
I've tried to do this before splitting into two setTimeout().
By doing the change the save doesn't work and a console error appears which I didn't undestand:
_com_liferay_object_web_internal_list_type_portlet_portlet_ListTypeDefinitionsPortlet_saveListTypeDefinition is not defined
    at HTMLFormElement.onsubmit `

Comment (alinedoleron) [reply modules/apps/object/object-web/src/main/resources/META-INF/resources/list_type_definitions/edit_list_type_definition.jsp:154]: I think it's better to show you on Monday.

**Comment by alinedoleron** [comment]
ci:test:relevant

**Review by miltonmc — APPROVED** [review]
LGTM!

**Comment by alinedoleron** [comment]
ci:forward

**Pull request closed without merging**